src/main/java/at/hannibal2/skyhanni/features/nether/kuudra/KuudraProfileViewerBlocker.kt

```diff
@@ -0,0 +1,25 @@
+package at.hannibal2.skyhanni.features.nether.kuudra
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.api.event.HandleEvent
+import at.hannibal2.skyhanni.data.ClickType
+import at.hannibal2.skyhanni.data.IslandType
+import at.hannibal2.skyhanni.events.entity.EntityClickEvent
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import net.minecraft.entity.player.EntityPlayer
+
+@SkyHanniModule
+object KuudraProfileViewerBlocker {
+
+    private val config get() = SkyHanniMod.feature.crimsonIsle
+
+    @HandleEvent(onlyOnIsland = IslandType.KUUDRA_ARENA)
+    fun onClickEntity(event: EntityClickEvent) {
+        if (!config.kuudraProfileViewer) return
+
+        if (event.clickedEntity !is EntityPlayer) return
```

Comment:
make sure they are no npc. we still wanna be able to interact with elle (or whatever other npcthere might be, idk i dont play kuudra)